### PR TITLE
[test] Fix deprecated `time_limit` syntax in checks

### PR DESCRIPTION
**Pull request by teojgo**

### CODE REVIEW

File: cscs-checks/apps/cpmd/cpmd_check.py [hunk 1]
```diff
@@ -21,7 +21,7 @@ def __init__(self, scale):
             self.valid_systems += ['dom:gpu']
         else:
             self.num_tasks = 16
-            self.time_limit = (0, 20, 0)
```
Comment (vkarak) [inline cscs-checks/apps/cpmd/cpmd_check.py]: Is this a realistic time limit? It looks like a workaround. Can you check please the execution time and then possibly remove this completely?

File: cscs-checks/microbenchmarks/spec-accel/spec.py [hunk 2]
```diff
@@ -42,7 +42,7 @@ def __init__(self, prg_envs):
 
         self.num_tasks = 1
         self.num_tasks_per_node = 1
-        self.time_limit = (0, 30, 0)
+        self.time_limit = '30m'
```
Comment (vkarak) [inline cscs-checks/microbenchmarks/spec-accel/spec.py]: Same here. I don't think they run for so long.

**Comment by vkarak** [comment]
Can you please check also the CI failures?

File: cscs-checks/microbenchmarks/spec-accel/spec.py (hunk 2 above)
Comment (teojgo) [reply cscs-checks/microbenchmarks/spec-accel/spec.py]: @vkarak this one needs this timelimit since the run takes longer.

**Comment by vkarak** [comment]
@jenkins-cscs retry none

**Pull request merged**